pages/content/shared/copy-script.html

```diff
@@ -12,23 +12,34 @@
     {% do doc.styles.addCssFile('/css/components/atoms/icon.css') %}
     {{ doc.styles.emit() }}
     html, body { background: none transparent; width: 24px; height: 24px; overflow: hidden; }
-    body { display: flex; align-items: center; margin: 0; min-height: 100vh; cursor: pointer; }
+    body { display: flex; align-items: center; justify-content: center; margin: 0; min-height: 100vh; cursor: pointer; transition: transform .3s;}
     svg { fill: #005af0; }
-    input { opacity: 0 }
+    input { opacity: 0; pointer-events: none; }
+    .copy-script-icon--default, .copy-script-icon--success { position: absolute; transition: transform .3s; }
+    .copy-script-icon--success { opacity: 0; }
+    .success { transform: translate3d(0, 2px, 0); }
+    .success .copy-script-icon--default { opacity: 0; transform: scale3d(0, 0.3, 0); }
+    .success .copy-script-icon--success { opacity: 1; }
   </style>
 </head>
-<body>
+<body id="copy-script">
   <svg style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
     <defs>
       {{ doc.icons.emit() }}
     </defs>
   </svg>
-  <div class="ap-a-ico">
+  <div class="ap-a-ico copy-script-icon--default">
     {% do doc.icons.useIcon('/icons/clipboard.svg') %}
     <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#clipboard"></use></svg>
   </div>
+  <div class="ap-a-ico copy-script-icon--success">
+    {% do doc.icons.useIcon('/icons/checkmark.svg') %}
+    <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#checkmark"></use></svg>
+  </div>
   <input id="script-tag">
   <script>
+    const animationDuration = 1200;
+    const copyScript = document.getElementById('copy-script');
```

Comment:
Use `document.body`. This makes the ID as well as the extra variable unnecessary.